subresource-loading.bs

```diff
@@ -55,41 +55,291 @@ Note: This specification is under construction. See
+    the string "`webbundle`", <a spec="html">the script's type</a> is
+    "`webbundle`".
+- Insert the following case to <a spec="html">prepare a script</a> step 24.6:
+
+  - "`webbundle`":
+
+    1. [=Queue a task=] to [=fire an event=] named `error` at the element, and
+       return.
+
+       Note: `<script type="webbundle" src=...>` is not supported. There are no
+       specific requirements for the error handling here, so currently an
+       `error` event is fired similarly to the case of an empty `src` attribute.
+
+- Insert the following case to <a spec="html">prepare a script</a> step 25.2:
+
+  - "`webbundle`":
+
+    1. [=Prepare a web bundle=] given <var ignore>element</var>,
+       <var ignore>source text</var> and <var ignore>base URL</var>.
+
+- Insert the following case to <a spec="html">prepare a script</a> step 26:
+
+  - If <a spec="html">the script's type</a> is "`webbundle`":
+
+    1. Assert: <a spec="html">the script is ready</a>.
+    1. [=In parallel=], [=process events for a web bundle=] given
+       <var ignore>element</var>.
+
+NOTE: CSPs are applied to inline web bundles at Step 13 of
+<a spec="html">prepare a script</a>, just like applied to classic/module
+scripts.
+
+To <dfn>prepare a web bundle</dfn>, given an {{HTMLScriptElement}} |element|, a
+[=string=] |sourceText| and a [=URL=] |baseURL|:
+
+1. Let |parse result| be the result of [=parse a web bundle string=] given
+   |sourceText| and |baseURL|.
+1. If this throws an exception:
+   1. Set [=the script's result=] to a new [=web bundle result=] with its [=web
+      bundle result/registration=] is null and its [=web bundle result/error to
+      rethrow=] is the exception thrown.
+   1. <a spec="html">The script is ready</a>.
+   1. Return.
+1. Let |document| be |element|'s <a spec="html">node document</a>.
+1. Set |fetch entry| to null.
+1. For each |r| in |document|'s [=Document/web bundle fetch entry list=]:
+
+   1. If |r|'s [=web bundle fetch entry/source=] is |parse result|'s [=web
+      bundle parse result/source=] and |r|'s [=web bundle fetch
+      entry/credentials=] is |parse result|'s [=web bundle parse
+      result/credentials=], then:
+
+      1. If |r| is not [=web bundle fetch entry/used by a registration=] in
+         |document|, then set |fetch entry| to |r|.
+
+         NOTE: This implies that another script element whose [=the script's
+         result=]'s [=web bundle result/registration=]'s [=web bundle
+         registration/fetch entry=] is |r| was removed. This is to ensure that
+         [=web bundle fetch entries=] are not destructed and re-fetched when
+         {{HTMLScriptElement}}s with the same web bundle [=web bundle fetch
+         entry/source=] and [=web bundle fetch entry/credentials=] are removed
+         and added.
+
+      1. Otherwise, then a user agent MAY set |fetch entry| to |r|.
+
+         ISSUE: Should this be "MUST set", "MAY set", or "MUST NOT set"?
```

Comment:
Let's omit this "Otherwise, then a user agent..." clause (That's effectively equivalent with "MUST NOT set").
As you noticed, Chromium doesn't do this optimization for this minor case, where I don't think there is a use case.

Let's keep the algorithm simple at the moment unless an optimization is needed in a real use case.

Reply:
Sounds good. Removed.